### PR TITLE
Promote Charset Mismatch Passive Scanner to Release

### PR DESCRIPTION
**Pull request by kingthorin**
Fixes zaproxy/zaproxy#4460

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Comment by kingthorin** [comment]
Unrelated to the promo, but is something up with `containsNameLoadedWithKey`. Even once the keys were removed from beta, I'd missed changing the base key name constant in the UnitTest and the unittests weren't failing? I even double checked by purposefully setting the wrong name....

File: addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/CharsetMismatchScannerUnitTest.java [hunk 1]
```diff
@@ -28,10 +28,12 @@
 import org.parosproxy.paros.core.scanner.Plugin.AlertThreshold;
 import org.parosproxy.paros.network.HttpMalformedHeaderException;
 import org.parosproxy.paros.network.HttpMessage;
+import org.zaproxy.zap.extension.pscanrules.CharsetMismatchScanner;
+import org.zaproxy.zap.extension.pscanrules.PassiveScannerTest;
```
Comment (thc202) [inline addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/CharsetMismatchScannerUnitTest.java]: These two imports can be removed.

**Comment by kingthorin** [comment]
Done

**Comment by thc202** [comment]
Thanks!

**Comment by thc202** [comment]
The issue mentioned earlier (https://github.com/zaproxy/zap-extensions/pull/2005#issuecomment-484306917) should be fixed with #2010.

**Pull request merged**